tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -15,12 +15,19 @@ public class VendoredDependency
     {
         private const string AutoGeneratedMessage = @"//------------------------------------------------------------------------------
 // <auto-generated />
-// This file was automatically generated by the UpdateVendors tool.
+// This file was automatically generated by the UpdateVendoredCode tool.
 //------------------------------------------------------------------------------
 ";
 
         static VendoredDependency()
         {
+            Add(
+                libraryName: "Microsoft.OpenApi",
+                version: "1.6.22",
+                downloadUrl: "https://github.com/microsoft/OpenAPI.NET/archive/1.6.22.zip",
+                pathToSrc: new[] { "OpenAPI.NET-1.6.22", "src", "Microsoft.OpenApi" },
+                transform: filePath => RewriteCsFileWithStandardTransform(filePath, originalNamespace: "Microsoft.OpenApi"));
```

Comment:
You should add the additional changes you need to make in here as additional transforms

Reply:
ooooh this is how it works